testsuite/gnat2goto/tests/incomplete_dec_priv/test.out

```diff
@@ -0,0 +1,11 @@
+Standard_Output from gnat2goto incomplete_dec_priv:
```

Comment:
I don't think the exact wording of this print-out is necessary to be included in the expected output. If nothing else it would be inconvenient to maintain.

Reply:
I am not sure what to suggest here.  The test suite identifies the source of the warning/error message and follows it with the text.  Once private_types are handled the warning/error messages will not appear.